docs/sources/tempo/configuration/_index.md

```diff
@@ -379,6 +379,9 @@ metrics_generator:
         # A list of labels that will be added to all generated metrics.
         [external_labels: <map>]
 
+        # If set, the tenant ID will added as label with the given label name to all generated metrics.
+        [inject_tenant_id_as: <string>]
```

Comment:
I'll be honest, I didn't really find a nice and concise way to describe this setting. Suggestion are welcome 😅

Also feedback about the name of the setting itself is welcome. It's a bit verbose now imo.

Reply:
I like it 👍

Reply:
Even if it's long, it's clear what it does.